### PR TITLE
LPD-22136 [Migration from Poshi to Playwright] Test relationship field autocomplete

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPD-22136;

About the **expect** parts with and without backspace:
- If there is no interaction with the search input, we have this situation:
regularMenuItem

- If there is some content into the search input:
altMenuItem

This variation of the tags causes these slight changes in the locator, during **expect** function.

### CODE REVIEW

File: modules/test/playwright/tests/object-web/objectEntries.spec.ts [hunk 1]
```diff
@@ -111,6 +111,89 @@ test.describe('Manage object entries through page templates', () => {
 		);
 	});
 
+	test('can view all entries related to an object in the relationship field using autocomplete', async ({
```
Comment (danvb112) [inline modules/test/playwright/tests/object-web/objectEntries.spec.ts:114]: Hey @p-albuquerque, can you move this test to inside of `test.describe` => **Manage object entries through View Object Entries**

**Review by danvb112 — CHANGES_REQUESTED** [review]
Hey @p-albuquerque, Great job with this test! 


I have some suggestions and a question: You shouldn't create a commit removing the poshi `ObjectPortlet#CanSearchPartialOnRelationshipField `test since we are now doing this using playwright?

**Comment by p-albuquerque** [comment]
ci:test:sf

**Review by danvb112 — APPROVED** [review]
LGTM! :tada:

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
ci:forward

**Pull request closed without merging**